### ISSUE TITLE
Include PhpStorm-related .phpstorm.meta.php changes in release version

### ISSUE THREAD

**Issue by wbars** [mockery/mockery#1181]
Hey! In 2021 We provided commit in master branch that improves codeinsight of mock objects in PhpStorm https://github.com/mockery/mockery/commit/905854e05285ae885dad8297807c4eaabdf3e7ed, but it seems it still doesn't present in latest Mockery release. We've created an [issue](https://youtrack.jetbrains.com/issue/WI-66818/Mockery-methods-should-treat-string-arguments-as-class-FQNs) to duplicate same behaviour in PhpStorm for old versions of Mockery. Can you please include this commit in new releases of Mockery?

**Comment by leonakao** [comment]
Do we have a expected release date?

**Comment by ghostwriter** [comment]
This issue has been resolved via #1126, on the `master` branch.

There is no exact expected release date. However, you may use the branch alias `dev-master`.

```BASH
composer require --dev mockery/mockery:dev-master
```

```JSON
{
    "require-dev": {
        "mockery/mockery": "dev-master"
    }
}
```
